src/midisequencer.cpp

```diff
@@ -79,8 +79,9 @@ namespace midisequencer{
         if (position != messages.begin()) {
             position--;
         }
-        return position;
+        return loop_position;
```

Comment:
oh, my bad. I broke this when I did the chrono refactor. This was ``loop_position->time`` before :/

Reply:
btw do we even need the rest of the function? ``position `` is a local variable and dead code.